src/cluster/src/install.rs

```diff
@@ -457,6 +485,29 @@ pub struct ClusterInstaller {
 }
 
 impl ClusterInstaller {
+    /// Creates a `ClusterInstaller` with default installation settings
+    ///
+    /// This will fail if there is trouble connecting to Kubernetes or
+    /// if the `helm` executable is not locally installed.
+    ///
+    /// To customize the installation settings, use the [`builder()`]
+    /// function to create a [`ClusterInstallerBuilder`].
+    ///
+    /// # Example
+    ///
+    /// ```no_run
+    /// use fluvio_cluster::ClusterInstaller;
+    /// let installer = ClusterInstaller::new().unwrap();
+    /// async_std::task::block_on(installer.install_fluvio()).unwrap();
```

Comment:
why use of `block_on` since this is not async fn

Reply:
`ClusterInstaller::install_fluvio` is an async function, so it needs the `block_on` to drive it